### DISCUSSION TITLE
demo source code

### DISCUSSION THREAD

**Discussion by zetoun17fr** [ruffle-rs/ruffle#2424] [Q&A] [answered]
Where can i find the source code of the [demo page](https://ruffle.rs/demo) ? unpacked version index.js and 662.index.js

my swf files works well on the demo page but dont work with the chrome plugin nor the self hosted version
"Serious error occurred loading SWF file: TypeError: JSON.stringify is not a function"

thanks

**Comment by torokati44** [comment, marked as answer]
It's in a fairly logical place in the repo: https://github.com/ruffle-rs/ruffle/tree/master/web/packages/demo/www

**Comment by zetoun17fr** [comment]
problem solved, 
i had to remove old javascript libraries (mootools) and old 'onload' events
